lib/loader.js

```diff
@@ -70,7 +70,7 @@ function createBot (options = {}) {
   options.loadInternalPlugins = options.loadInternalPlugins ?? true
   options.client = options.client ?? null
   options.brand = options.brand ?? 'vanilla'
-  options.respawn = options.respawn ?? true
+  options.autoRespawn = options.autoRespawn ?? options.autoRespawn ?? true
```

Comment:
Incorrect logic

Also, the doc seems already pretty clear. Is there anyone that's been confused by this?

Reply:
oops, fixed